tests/tools/onert_train/src/args.cc

```diff
@@ -199,6 +200,40 @@ void Args::Initialize(void)
     }
   };
 
+  auto check_range = [](const std::initializer_list<int> arg_list, const int value,
+                        const std::string &arg_name = "") {
+    for (const auto &i : arg_list)
+    {
+      if (i == value)
+        return;
+    }
+    throw std::runtime_error{arg_name + " " + std::to_string(value) + " is unsupported argument"};
```

Comment:
IMO, it would be better to print error message instead of throwing an error.

```
E: Fail to run by runtime error:loss 5 is unsupported argument
```

Reply:
updated:) 
I make it to print an error message to std::cerr and end with `exit(1)`.  
Is it right that you intended?